### PR TITLE
Create create_hdd_x86_64_uefi testsuite to use autoyast

### PR DESCRIPTION
**Pull request by wangqianzou**
Poo: https://progress.opensuse.org/issues/111225
Poo: https://progress.opensuse.org/issues/111356
Create create_hdd_x86_64_uefi to use autoyast

- Related ticket: https://progress.opensuse.org/issues/111225; https://progress.opensuse.org/issues/111356
- Needles: N/A
- Verification run:
   - https://openqa.suse.de/tests/8820641#
   - https://openqa.suse.de/tests/8825852#

### CODE REVIEW

**Review by rfan1 — APPROVED** [review]
LGTM

**Comment by rfan1** [comment]
Let us wait for experts' comments from yast team before merge this PR

**Comment by rfan1** [comment]
> Let us wait for experts' comments from yast team before merging this PR

**Review by rfan1 — COMMENTED** [review]
@wangqianzou 
BTW, Can we use the same yaml_schedule file as existing ones?

**Comment by jknphy** [comment]
I would remove the word `migration` unless you plan to replace existing test suite in your job group by this one.
Otherwise is not a migration it is a new test suite added in Functional group.
I would recommend the first option, groups will become more stable not sharing autoyast jobs, at the cost of repeating the AutoYaST installation in several groups.
In YaST squad we are doing that in maintenance, to have our own images because it makes it more solid, but in product validation there are too many and Core squad is responsible for that, so I don't recommend to add yet another one, in Functional group we can migrate to avoid repeat over and over steps of the installation but for new image creation I would recommend that you setup in your job group unless you check with Core if they are interested in take responsibility and review that test suite. Perhaps your plan was to have it in your group and I got confused when I saw the path of of one of the files which contains `/functional/, but just in case :)
Other than that LGTM, if covers your needs, I didn't see verification of the chained jobs and you are publishing the image.

File: schedule/functional/autoyast_create_hdd/autoyast_create_hdd_gnome_uefi.yaml
Comment (jknphy) [inline schedule/functional/autoyast_create_hdd/autoyast_create_hdd_gnome_uefi.yaml:6]: I guess the `uefi` part should be after x86_64, or the architecture: `create_hdd/create_hdd_%DESKTOP%_%ARCH%_uefi.xml`

**Comment by wangqianzou** [comment]
> I would remove the word `migration` unless you plan to replace existing test suite in your job group by this one. Otherwise is not a migration it is a new test suite added in Functional group. I would recommend the first option, groups will become more stable not sharing autoyast jobs, at the cost of repeating the AutoYaST installation in several groups. In YaST squad we are doing that in maintenance, to have our own images because it makes it more solid, but in product validation there are too many and Core squad is responsible for that, so I don't recommend to add yet another one, in Functional group we can migrate to avoid repeat over and over steps of the installation but for new image creation I would recommend that you setup in your job group unless you check with Core if they are interested in take responsibility and review that test suite. Perhaps your plan was to have it in your group and I got confused when I saw the path of of one of the files which contains `/functional/, but just in case :) Other than that LGTM, if covers your needs, I didn't see verification of the chained jobs and you are publishing the image.

Thanks for you suggestion, Security team will create security schedule directory to implement it.

File: schedule/security/create_hdd_autoyast/autoyast_create_hdd_x86_64_uefi.yaml
Comment (rfan1) [inline schedule/security/create_hdd_autoyast/autoyast_create_hdd_x86_64_uefi.yaml:3]: Modify this description,
currently, it is target for our security tests only

Comment (rfan1) [reply schedule/security/create_hdd_autoyast/autoyast_create_hdd_x86_64_uefi.yaml:3]: create_hdd_x86_64_uefi.yaml

**Comment by rfan1** [comment]
@wangqianzou 
Could you please add the chained job which use the publish qcow2 images [both hdd and uefi_vars]

**Review by Amrysliu — APPROVED** [review]

**Review by STARRY-S — APPROVED** [review]
LGTM :)

**Comment by jknphy** [comment]
LGTM

**Pull request merged**
